### PR TITLE
For #18742 - Resize the top site item sizes

### LINKED ISSUE #18742: Resize the top site items
Figma: https://www.figma.com/file/lfgj15zbNkL9Pp4hdZGJgp/Proton-Mobile-UI?node-id=809%3A1

| Element | Theme | Current | Proposed |
| --- | --- | --- | --- |
| Shortcut tile background | Both | 40×40 | 60×60 |
| Shortcut tile favicon | Both | 32×32 | 36×36 |

### PR DESCRIPTION
**Pull request by gabrielluong**
Fixes #18742 

Screen Shot 2021-04-01 at 12 10 43 PM

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Review by topotropic — COMMENTED** [review]
Can you help me understand why the values here don't match Bram's original spec (favicon size: 36x36 and background: 60x60)? Thank you!

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -206,9 +206,9 @@
     <dimen name="account_settings_device_name_min_height">48dp</dimen>
 
     <!-- Top sites  -->
-    <dimen name="top_sites_favicon_size">40dp</dimen>
-    <dimen name="top_sites_favicon_padding">4dp</dimen>
-    <dimen name="top_sites_item_size">64dp</dimen>
+    <dimen name="top_sites_favicon_size">60dp</dimen>
+    <dimen name="top_sites_favicon_padding">12dp</dimen>
```
Comment (gabrielluong) [inline app/src/main/res/values/dimens.xml:210]: This is the padding to get the favicon to be 36x36.

Screen Shot 2021-04-08 at 4 21 52 PM

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -206,9 +206,9 @@
...
+    <dimen name="top_sites_item_size">84dp</dimen>
```
Comment (gabrielluong) [inline app/src/main/res/values/dimens.xml:211]: This is the height. We will need to adjust this in a separate issue when we make the label be more than 1 line.

Screen Shot 2021-04-08 at 4 22 40 PM

Comment (topotropic) [reply app/src/main/res/values/dimens.xml:211]: Thank you! This was really helpful,appreciated.

**Review by topotropic — APPROVED** [review]

**Review by jonalmeida — APPROVED** [review]

**Pull request merged**